bindings/node/index.d.ts

```diff
@@ -179,7 +179,8 @@ export interface KMSProviders {
          * If present, an access token to authenticate with Azure.
          */
         accessToken: string;
-      };
```

Comment:
We have new errors exported as well: 
- MongoCryptAzureKMSRequestError,
- MongoCryptKMSRequestNetworkTimeoutError